userguide/system.rst

```diff
@@ -1325,15 +1325,15 @@ the storage devices will not be accessible by older versions of
 Updates and Trains
 ~~~~~~~~~~~~~~~~~~
 
-%brand% uses signed update files. This provides flexibility in deciding
-when to upgrade the system with patches, new drivers, or new features.
-It also allows "test driving" an upcoming release. Combined with boot
-environments, new features or system patches can be tested while
-maintaining the ability to revert to a previous version of the operating
-system, using the instructions in :ref:`If Something Goes Wrong`.
-Digitally signed update files eliminate the need to manually download
-both an upgrade file and the associated checksum to verify file
-integrity.
+%brand% uses signed update files downloaded from an encrypted source.
+This provides flexibility in deciding when to upgrade the system with
+patches, new drivers, or new features. It also allows "test driving" an
+upcoming release. Combined with boot environments, new features or
```

Comment:
> Combined with boot environments, new features or system patches can be tested while maintaining the ability to revert to a previous version of the operating system, using the instructions in Digitally signed update files eliminate the need to manually download :ref:`If Something Goes Wrong`.

This sentence took me a few reads to understand it. This could, perhaps, be split into 2 sentences. Take the suggestion as you will, but maybe something along the lines of:

> Boot environments, new features, or system patches can be tested. Revert to a previous version of the operating system by using the instructions in :ref:`If Something Goes Wrong`.